linalg/blockoperator.cpp

```diff
@@ -238,6 +237,7 @@ BlockDiagonalPreconditioner::~BlockDiagonalPreconditioner()
    }
 }
 
+//-----------------------------------------------------------------------
```

Comment:
?

Reply:
removed in https://github.com/mfem/mfem/pull/4675/commits/ea041467ae1ad6d007ea2a6e7880fbfa448f50a7